src/util/getThoughtSpacing.ts

```diff
@@ -0,0 +1,14 @@
+/** Get thought space. */
+export const getThoughtSpacing = (distance: number, depth: number): number => {
```

Comment:
Since `getThoughtSpacing` is only used in one place, it should be defined in that module. Creating a separate util function unnecessarily increases the surface area of the internal API. Unless it is a very large implementation, it's also better to co-locate functions so that a developer can look at the implementation without leaving the file.

Reply:
Makes sense.

Reply:
Also fixed.